tests/EndToEnd/TwoStepPaymentTerminalTest.php

```diff
@@ -25,23 +25,23 @@ public function testPaytureInPayApi(): void
         );
         $sessionId = $response->getSessionId();
 
-        $response = $this->getTerminal()->payStatus($orderId);
+        $response = $this->getTerminal()->getState($orderId);
```

Comment:
We should keep e2e tests with `payStatus` as long as we provide this method in the API. we can remove this tests alongside with the method in next major version published. Until then I think we should duplicate test suite

Reply:
Ok I understand. Will fix it.